src/azure-cli-core/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 unreleased
 +++++++++++++++++++
 * fixes issue where `three_state_flag` would not work correctly if custom labels were used.
+* remove Container Service from profiles
```

Comment:
Since this change isn't user facing, there doesn't need to be anything in HISTORY.rst.